gcx/styles/src/partials/head-icons.hbs

```diff
@@ -1,2 +1,2 @@
     <link rel="icon" href="https://www.datastax.com/favicon.ico" type="image/x-icon">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
\ No newline at end of file
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
I added a blank line at the end of the file to make the HTML output more organized.